examples/audit-croatia/step-8.html

```diff
@@ -4,4 +4,5 @@
     <p>This button is the middle one of the three buttons available above the first table.</p>
     <p>After clicking on the button you will be prompted with another captcha. Fill it in correctly and then you should
         be presented with a PDF document.</p>
+    <img src="{image1}" alt="image1">
```

Comment:
When you presented the workflow at the meeting on Friday, I've seen that you had problems with big images on the bigger screen. I'd recommend you to
create a class:
```css
.inline-img {
  max-width: 500px;
  margin: 0 auto;
}
```

```html
<img class="inline-img" src="{image1}" alt="image1">
```

That will shrink the width of the image automatically up to 500px and set the image in the center. I hope it works. 😆